docs/user/filesystem.md

```diff
@@ -1019,7 +1019,8 @@ libpil4dfs intercepting summary for ops on DFS:
 
 [op_sum ]  5003
 ```
-
+### The reported number of interception
+libioil intercepts streaming IO functions, fread, fwrite, etc. It issues IO request via dfs API immediately for each call. libpil4dfs only intercepts low level API read/write. Streaming APIs like fread, fwrite, fprintf etc. are not explicitly intercepted. The default streaming IO buffer size is 8KB (may depend on OS and glibc version). The low level API read() is called by streaming IO API in case read buffer is empty/depleted. write() is called when the write buffer is full or fclose() is called. Consequently, the number of IO interception reported by libioil and libpil4dfs could be different.
```

Comment:
```suggestion
libioil intercepts streaming IO functions like fread, fwrite, etc. It issues IO request via dfs API immediately for each call. libpil4dfs only intercepts low level API read/write. Streaming APIs like fread, fwrite, fprintf, etc. are not explicitly intercepted. The default streaming IO buffer size is 8KB (may depend on OS and glibc version). The low level API read() is called by streaming IO API in case read buffer is empty/depleted. write() is called when the write buffer is full or fclose() is called. Consequently, the amount of IO interception reported by libioil and libpil4dfs could be different.
```

Reply:
Thank you very much! Fixed as you suggested.